### ISSUE TITLE
x-axis labels disappear

### ISSUE THREAD

**Issue by woodwater** [getredash/redash#3965]
I tried to generate a report by 7.0 version and found that when i chose the bar view, the x axis labels disappeared. They only appear when I make the report smaller to show the y-axis from (-500,500). 
![image](https://user-images.githubusercontent.com/2803255/60946265-97e95900-a30f-11e9-9b20-2f44aa48afc9.png)
I tried to adjust the y-axis range, but it didnt help. 
![image](https://user-images.githubusercontent.com/2803255/60946284-a6d00b80-a30f-11e9-8d47-67684e92a79b.png)

**Comment by kravets-levko** [comment]
Seems it's Plotly issue: it hides x-axis if there is no enough space for it. Can you try to build the same chart, but with shorter category names and tell if x-axis stays visible?

**Comment by woodwater** [comment]
Thanks. You are right. 

![image](https://user-images.githubusercontent.com/2803255/60954584-100c4a80-a321-11e9-83ca-b09836be71e4.png)
It works if I changed it to another column. 
> Seems it's Plotly issue: it hides x-axis if there is no enough space for it. Can you try to build the same chart, but with shorter category names and tell if x-axis stays visible?

**Comment by kravets-levko** [comment]
Yep. Not sure if we can fix it somehow (and what could be a good fix for this?)

**Comment by woodwater** [comment]
Thanks for your response. 
Actually it works if we don't choose the bar format.
![image](https://user-images.githubusercontent.com/2803255/60960187-5a92c480-a32b-11e9-8ef7-b91cb97306d0.png)
